[2024-05-21T15:16:25.872+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-21T15:16:25.883+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: idealista-pipeline.send_to_kafka manual__2024-05-21T15:15:47.516620+00:00 [queued]>
[2024-05-21T15:16:25.888+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: idealista-pipeline.send_to_kafka manual__2024-05-21T15:15:47.516620+00:00 [queued]>
[2024-05-21T15:16:25.889+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-21T15:16:25.896+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): send_to_kafka> on 2024-05-21 15:15:47.516620+00:00
[2024-05-21T15:16:25.900+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=460) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-21T15:16:25.901+0000] {standard_task_runner.py:63} INFO - Started process 461 to run task
[2024-05-21T15:16:25.901+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'idealista-pipeline', 'send_to_kafka', 'manual__2024-05-21T15:15:47.516620+00:00', '--job-id', '299', '--raw', '--subdir', 'DAGS_FOLDER/idealista_dag.py', '--cfg-path', '/tmp/tmpcwci18ok']
[2024-05-21T15:16:25.902+0000] {standard_task_runner.py:91} INFO - Job 299: Subtask send_to_kafka
[2024-05-21T15:16:25.924+0000] {task_command.py:426} INFO - Running <TaskInstance: idealista-pipeline.send_to_kafka manual__2024-05-21T15:15:47.516620+00:00 [running]> on host 05882356329d
[2024-05-21T15:16:25.976+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='shv' AIRFLOW_CTX_DAG_ID='idealista-pipeline' AIRFLOW_CTX_TASK_ID='send_to_kafka' AIRFLOW_CTX_EXECUTION_DATE='2024-05-21T15:15:47.516620+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-21T15:15:47.516620+00:00'
[2024-05-21T15:16:25.977+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-21T15:16:25.988+0000] {minio.py:45} INFO - Minio successfully connected.
[2024-05-21T15:16:25.988+0000] {minio.py:93} INFO - Start reading unprocessed data
[2024-05-21T15:16:25.989+0000] {minio.py:119} INFO - Start reading data from object name processed_files
[2024-05-21T15:16:26.007+0000] {minio.py:119} INFO - Start reading data from object name response2024_05_21_p_2.json
[2024-05-21T15:16:26.026+0000] {kafka.py:62} WARNING - Failed to send message to kafka topic. Error Expecting property name enclosed in double quotes: line 1 column 2 (char 1)
[2024-05-21T15:16:26.027+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-21T15:16:26.027+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-21T15:16:26.031+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=idealista-pipeline, task_id=send_to_kafka, execution_date=20240521T151547, start_date=20240521T151625, end_date=20240521T151626
[2024-05-21T15:16:26.074+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-21T15:16:26.082+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-21T15:16:26.085+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
